lib/config.js

```diff
@@ -1,39 +1,32 @@
-// All imports in this file must have been installed from the trimet-mod-otp
-// project.
-import React from 'react'
-
 /**
- * To perform some custom operations, create a file like this one and reference
- * it in the config.yml entry `jsConfigFile`.
+ * To perform some custom operations, create a file like this one and specify it
+ * during build/start to override this default version of the file:
+ *    yarn start --env.JS_CONFIG=/path/to/config.js
  *
  * Caveats: This file cannot import the following:
  * - other files
  * - modules not installed in the node_modules of this project
- *
+ * - see config-example.js for more options on icon customization
+ */
+
+// All imports in this file must have been installed from the trimet-mod-otp
+// project.
+import React from 'react'
+
+// Import the base icon set use/customize.
+import { TriMetLegIcon, TriMetModeIcon } from '@opentripplanner/icons'
+
+/**
  * @param  {Object} otpConfig The contents of the yml config file as json.
+ * @param  {Object} customIcons Mapping of mode to custom icon component
  * @return  This function must return an object and can have one of the
```

Comment:
Refactor this comment to note a new requirement for LegIcon and ModeIcon.

Reply:
If you wanted to pass a LegIcon and ModeIcon, you would import hem first then pass them to the return value.

Reply:
In `lib/app.js`, an error is now thrown if LegIcon or ModeIcon are not defined, so I am asking for this comment here to be changed from saying 

> This function ... can have one of the following 

to noting which values are required and which ones are optional.

Reply:
Updated as of 5ea9f8cc8d313c869408c8101a200c21a68eea1f